webcentral/doc/06_sphinx/source/general/getting_started.md

```diff
@@ -1,33 +1,36 @@
-
-Furthermore `node.js` is needed. It can be downloaded from the official website: https://nodejs.org/en/download/package-manager. Please install the latest LTS-version.
-
-After that please clone the repo to a local location of your choice. 
-The application can be executed by doing the following steps:
-1. Create a .env-file from the the .env.example file.
-`
+```{note}
+We have tested the web application on linux, macOS and windows. However if you are not working on linux you need to install the `Bash`-shell, since bash-scripts are used as utilities and as a entry-point to the application. One way to install Bash is when installing `git under windows <https://gitforwindows.org/>`. In the following guide, linux commands are used. Please execute these commands in your installed Bash-shell. 
+```
+1. Please clone the repository to a local location of your choice. 
+2. Create a .env-file from the the .env.example file.
+```
   cp .env.example .env
-`
-2. Execute the command `npm install` to install the `node.js`-dependencies these are used to transpile the `scss`-stylesheets into a bundled `css`-stylesheet.
-To build the development environment execute:
+```
+3. Execute the command `npm install` to install the `node.js`-dependencies these are used to transpile the `scss`-stylesheets into a bundled `css`-stylesheet.
+4. Build the development environment execute:
 ```
     ./run build_initial dev
 ```
-3. In the `Wissensplattform` images like logos of in the database included tools, are not located in the repository. These files have to be downloaded from the following link `https://tubcloud.tu-berlin.de/f/3546499069`. The `media`-folder has to be copied to `01_application/webcentral_app/`.
-3. Start the setup process by calling the run script with the argument `up_initial` and providing a database dump file.
+To interact with the web application a bash script with the name `run` is used. This script is used to interact with the application.  
+
+4. In the `Wissensplattform` images like logos of in the database included tools, are not located in the repository. These files have to be downloaded from the following link `https://tubcloud.tu-berlin.de/f/3546499069`. The `media`-folder has to be copied to `webcentral/src/media/`.
+
+5. In a last step the database needs to be populated with data. For that a database dump is located inside the repository under `postgres/webcentral_db_20240927_translation_use_cases_data_sufficiency.sql`.
+```{warning}
+Please note that the filename of the database dump may change. Please look for the latest file with the extension `.sql` in the `postgres/`-folder.
+```
+
+Start the setup process by calling the run script with the argument `up_initial` and providing a database dump file.
+```
```

Comment:
See previous comment

Reply:
I have removed the filename of the sql file and added a generic sql-filename in the `up_inital` command